changelogs.py

```diff
@@ -0,0 +1,38 @@
+import re
+import subprocess
+import sys
+from datetime import date
+import fileinput
+
+# Save new changelogs in temp.md
+changelogs_cmd = "semantic-release changelog"
+process = subprocess.Popen(changelogs_cmd.split(), stdout=subprocess.PIPE)
+output, error = process.communicate()
+output = "## [" + sys.argv[1] + "] - " + str(date.today()) + "\n"+output.decode("utf-8")
+
+file = open('temp.md', 'w')
+file.writelines(output)
+file.close()
+
+# Convert the changelogs to rst
+pandoc_cmd = "pandoc temp.md -f markdown_strict -t rst -o temp.rst"
+process = subprocess.Popen(pandoc_cmd.split(), stdout=subprocess.PIPE)
+output, error = process.communicate()
```

Comment:
Perhaps you should check the error status, or use `check_output` if you are only going to use the output.